pkg/webhooks/validation/server/server.go

```diff
@@ -210,6 +210,10 @@ func (wh *Webhook) validate(request *kube.AdmissionRequest) *kube.AdmissionRespo
 		return toAdmissionResponse(fmt.Errorf("cannot decode configuration: %v", err))
 	}
 
+	if request.Kind.Kind == "ServiceMeshExtension" {
+		return &kube.AdmissionResponse{Allowed: true, Warnings: []string{"ServiceMeshExtension is deprecated. Please use WasmPlugin instead."}}
```

Comment:
nit: Make this text a variable so it can be reused in tests.

Also, what about something like:

`ServiceMeshExtension is deprecated. It works now but will be removed in a future version. Please use WasmPlugin instead.`

Reply:
Done